.github/workflows/release.yml

```
@@ -46,10 +51,37 @@ jobs:
           echo "::set-output name=name::${GITHUB_REF#refs/*/}"
           echo "::set-output name=version::${GITHUB_REF##*/}"
         id: extract_branch
+      - name: Check npm version preid
+        run: |
+          HAS_PREID=$(node -p "'${{ steps.extract_branch.outputs.version }}'.startsWith('prerelease-') ? 1 : 0")
+          echo "::set-output name=has_preid::${HAS_PREID}"
+          if [ $HAS_PREID -eq 1 ]; then
+            PREID_NAME=$(node -p "const v = '${{ steps.extract_branch.outputs.version }}'.match(/^prerelease-(.+)$/); v ? v[1] : ''")
```

[nits] `^prerelease-` 以降の文字が欲しいのであれば趣向を変えて

```suggestion
            PREID_NAME=$(node -p "'${{ steps.extract_branch.outputs.version }}'.replace(/^prerelease-/, '')")
```

でもいいのかなと思いましたが、matchの方がマッチしたかどうか判定できるからmatchの方がいいかもですなあ。

これまた `steps.extract_branch.outputs.version` を環境変数に入れて

```sh
${VERSION#prerelease-}
```

としても`prerelease-`を除いた文字列が取れますがまあ、matchの方がいいのかなあ。

---

https://github.com/openameba/cwv-logger/pull/8/commits/f194e3de858772c9a55730dbfbbcf3e40cfa1c23
下のコメントと合わせて判定周りをshellで行うようにしました

matchでも良いですがここでチェックしているので少し冗長かなと思ったので修正しました:pray:
https://github.com/openameba/cwv-logger/pull/8/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R56